lib/node_modules/@stdlib/math/base/special/cexp/examples/index.js

```diff
@@ -21,24 +21,20 @@
 var Complex128 = require( '@stdlib/complex/float64' );
 var randu = require( '@stdlib/random/base/randu' );
 var round = require( '@stdlib/math/base/special/round' );
-var real = require( '@stdlib/complex/real' );
-var imag = require( '@stdlib/complex/imag' );
 var cexp = require( './../lib' );
 
 var re;
 var im;
 var z1;
 var z2;
-var o;
 var i;
 
 for ( i = 0; i < 100; i++ ) {
 	re = round( randu()*100.0 ) - 50.0;
```

Comment:
Same comment as in the README. Would you mind updating this example to use `discrete-uniform`?